src/utils/tile-validation-data.ts

```diff
@@ -0,0 +1,25 @@
+import type { TileValidationData } from "./types";
```

Comment:
I suggest moving this file in `tile-validation` folder.

Reply:
Done